docs/manuals/deployment/linux/redhat.md

```diff
@@ -0,0 +1,190 @@
+
+Gui(page="# Getting started with *Taipy*").run()
+```
+
+This would be placed in a file called _app.py_.<br>
+You need to create a _requirements.txt_ file that contains:
+```
+taipy
+```
+
+On your local machine, start the application by doing:
+```console
+$python3.8 -m pip install -r requirements.txt
+Collecting taipy
+<-- Truncate -->
+Successfully installed taipy
+$python3.8 app.py
+ * Server starting on http://127.0.0.1:5000
+ * Serving Flask app 'Taipy' (lazy loading)
+ * Environment: production
+   WARNING: This is a development server. Do not use it in a production deployment.
+   Use a production WSGI server instead.
+ * Debug mode: on
+<-- Truncate -->
+```
+
+The application is running locally, you can access it with the browser on the URL [http://127.0.0.1:5000/](http://127.0.0.1:5000/).
+
+!!! note
+    The message:
+    ```
+    WARNING: This is a development server. Do not use it in a production deployment.
+    Use a production WSGI server instead.
+    ```
+    Is provided by Flask because the way of exposing an application on the Internet and
+    developping an application locally is not the same, mainly for security and reliability reasons.
+    This message will disappear when using the web server.
+
+## Prepare the application for deployment
+
+Deploying your application on a remote environment needs a little bit of configuration.
+
+By default, Taipy applications run in Debug mode. Before deploying your application to the Internet,
+you should turn off the Debug mode by setting the _debug_ parameter or the `(Gui.)run()^` to False. <br>
+You must also inform Taipy not to run the application server on its own but rather delegate the execution
+by setting the parameter _run_server_ to False.<br>
+The name of the variable where the Web application is stored is used in the _uWSGI_ configuration:
+this allows the Web server to load the Web application:
+```
+from taipy import Gui
+
+gui_service = Gui(page="# Getting started with *Taipy*")
+web_app = gui_service.run(debug=False, run_server=False)
+```
+In our example, we store this application in the variable _web_app_ (see line 3)
+
+Make sure you upload this code on your targeted machine and install your dependencies with _pip_.
+
+!!! important
+    The entrypoint filename and the app variable name are important for the proper configuration of
+    the _uWSGI_ Web application server. Please, keep them as is or adapt the configuration.
+
+
+## uWSGI application server
+
+To expose your application over the Internet, you must use _uWSGI_ instead of Flask as the application server.
+You would then leverage Nginx to expose the application.
+
+_uWSGI_ can be started manually. But, generally, it's better to start the application automatically when the machine
+starts. To order to do that, you should use [Systemd](https://systemd.io/) which is installed by default on _Ubuntu_.
+
+Beside your application code, run the following command to generate an adapted file for _Systemd_:
```

Comment:
What does "Beside your application code" mean, here?

Reply:
"In the same directory, then the _app.py_, run the following command to generate an adapted file for _Systemd_"
Is it better?